src/rgw/rgw_lua_background.h

```diff
@@ -167,7 +166,6 @@ class Background : public RGWRealmReloader::Pauser {
 public:
   Background(rgw::sal::Driver* driver,
       CephContext* cct,
-      const std::string& luarocks_path,
```

Comment:
please remove the path from: https://github.com/ceph/ceph/pull/52718/files#diff-24cfbb1c5be3bc1eca1dc5ce0453821da744dad157e4821b4528f10515b47780L581